src/app/app.component.test.js

```diff
@@ -0,0 +1,14 @@
+import { AppComponent, NAVIGATION_ROUTES } from './app.component';
+
+describe('App', () => {
+    describe('#constructor', () => {
+        it('should be defined', () => {
```

Comment:
I suppose we can remove this test, because in next tests we check this implicitly